src/main/java/school/lemon/changerequest/java/introduction/hw2/Task5.java

```diff
@@ -2,12 +2,13 @@
 
 public class Task5 {
     public static boolean commondEnd(int[] a, int[] b) {
-        int lengthA = a.length;
-        int lengthB = b.length;
 
-        if (lengthA >= 1 && lengthB >= 1) {
-            return a[0] == b[0] || a[a.length - 1] == b[b.length - 1] || a[0] == b[b.length - 1] || a[a.length - 1] == b[0];
-        }
+
+        if (a[0] == b[0] || a[0] == b[b.length - 1] || a[a.length - 1] == b[0])
+            return true;
+
         return false;
+
+
```

Comment:
1. you don't need `if` clause, as result of your expression is `boolean` - you can just return it.
2. you should cover one more case - last `a` is equal to last `b`.